src/main/java/frc/robot/Constants.java

```diff
@@ -57,7 +57,7 @@ public static final class FrontRight { // Module 1
 
         public static final double STEER_OFFSET =
             IS_COMP_BOT
-                ? -Math.toRadians(329.4937) // comp bot offset
```

Comment:
Why are you setting offsets in a feature?

Reply:
I wanted to test its driving, but modules needed calibration first. In hindsight, probably should have made a fix-branch first (like you said)